kong/router/transform.lua

```diff
@@ -546,6 +570,14 @@ local function get_priority(route)
 
   local match_weight = 0  -- 0x0ULL
 
+  if not is_empty_field(srcs) then
+    match_weight = match_weight + 1
+  end
+
+  if not is_empty_field(dsts) then
+    match_weight = match_weight + 1
+  end
```

Comment:
Why can `srcs` and `dsts` now increase `match_weight`?

Reply:
Now atc-router supports `srcs` and `dsts`, so if route has these rules we should increment its priority.